extensionsGallery-insider.json

```diff
@@ -153,6 +153,57 @@
 				],
 				"statistics": [],
 				"flags": "preview"
+			},
+			{
+				"extensionId": "13",
+				"extensionName": "ssmskeymap",
+				"displayName": "SSMS Keymap",
+				"shortDescription": "This extension ports popular SSMS keyboard shortcuts to SQL Operations Studio",
+				"publisher": {
+					"displayName":"Kevin Cunnane",
+					"publisherId": "kevcunnane",
+					"publisherName":"kevcunnane"
+				},
+				"versions": [
+					{
+						"version": "0.2.0",
+						"lastUpdated": "4/16/2018",
+						"assetUri": "",
+						"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+							{
+								"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+								"source": "https://github.com/kevcunnane/ssmskeymap/releases/download/0.2.0/ssmskeymap-0.2.0.vsix"
```

Comment:
does this popup the web browser to download, or auto-install?  If it is auto-install then we should probably host the VSIX in the Azure BLOB storage with the other extensions?  

If you're going for the community extension experience then you should use the download and manually install VSIX experience.

Reply:
I am going for the community experience on this. What is the syntax used for the "download and manually install" experience? I'm documenting this in a blog post so would be good to capture exactly what folks in the community need to do.